src/main/python/main.py

```diff
@@ -21,6 +21,9 @@
 # http://timlehr.com/python-exception-hooks-with-qt-message-box/
 from util import init_logger
 
+#Single instance app
+from single_application import QtSingleApplication
```

Comment:
Please move the import below (right above the `app = QtSingleApplication` line) so it is not imported when using linux-recorder

Reply:
Done